src/firefly/js/tables/ui/BasicTable.jsx

```diff
@@ -92,15 +87,15 @@ export class BasicTable extends React.Component {
                     }
                 };
 
-        const colProps = pick(this.props, ['columns', 'data', 'selectable', 'selectInfoCls', 'tableStore']);
+        const colProps = pick(this.props, ['columns', 'data', 'selectable', 'selectInfoCls', 'tableStore', 'renderers']);
```

Comment:
I didn't know about 'pick' function and i think is very nice. Actually i went to loadash docs and find out couple of others like 'omitBy' which is also interesting for oposite case when you want to pick from an object leaving out (omitting) attributes from it.